docs/GettingStarted.rst

```diff
@@ -1695,6 +1695,34 @@ You can add a text message or not and then click on 'Post' button to share the f
 
 .. note:: If the upload is still in progress when you apply the attachments, it will continue in the background. An animation around the attachments counter badge indicates progress. Note that the post button will be disabled until all files are uploaded.
 
+Selecting destination folder for uploads
+``````````````````````````````````````````
+
+You can select destination folders for uploaded files from the new composer. 
+Once the files are uploaded, you can use the 'folder' icon to choose a destination folder where to store them.
+It is possible to select the storage for all uploaded files using the folder icon above all files or file per file separately.
+
+|image234|
+
+When you select the principal folder icon, all uploaded files will be stored in the selected folder.
+When you select a destination folder for one file, the information will be displayed and it is possible to undo the action using the 'X' icon
+
+|image236|
+
+If the added files are selected from existing uploads, it is not possible to change the destination folder. 
+A forbidden sign icon is displayed instead of the folder one for these files. A tooltip "Not available for selected files" is displayed when we mouse over this icon.
```

Comment:
Not sure that the sentence "A tooltip ..." is relevant. We are going too deep in the detail IMO, and does not bring interesting information to the user.

Reply:
Ok i deleted information about tooltips